docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+services:
+  modernuo:
+    build:
+      context: .
+      args:
+        OS: "${OS:-linux}"
+        ARCH: "${ARCH:-amd64}"
+    environment:
+      UO_DIRECTORY: ${UO_DIRECTORY:-/bin/UO}
+    ports:
+      - "2593:2593"
+      - "12000:12000"
+    volumes:
+      - "./Distribution/Saves:/bin/modernuo/Saves"
+      - "./Distribution/Configuration:/bin/modernuo/Configuration"
+      - "$UO_DIRECTORY:$UO_DIRECTORY"
```

Comment:
tl;dr my UO directory is something like `/Users/icculusc/Downloads/UO`, so that's what it is in the `modernuo.json` from running it on the metal, this maps it directly to that directory, this is a bad solution to be fair